flake.nix

```
@@ -11,15 +11,17 @@
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     pre-commit-hooks-nix.url = "github:cachix/pre-commit-hooks.nix";
   };
-  outputs = inputs @ { flake-parts, ... }:
+  outputs = inputs @ { flake-parts, dream2nix, ... }:
     flake-parts.lib.mkFlake
       {
         inherit inputs;
       }
       {
         imports = [
           ./nix/host
-          ./nix/packages
+          (import ./nix/packages {
+            inherit dream2nix;
+          })
```

```suggestion
          ./nix/packages
```
You can simply import `./nix/packages` and access `inputs.dream2nix` argument in the imported module.